src/server/logs/logs.go

```diff
@@ -479,9 +476,19 @@ func (a *adapter) publish(ctx context.Context, entry loki.Entry) (bool, error) {
 	if err := msg.Object.UnmarshalJSON([]byte(entry.Line)); err != nil {
 		log.Error(ctx, "failed to unmarshal json into protobuf Struct", zap.Error(err), zap.String("line", entry.Line))
 		msg.Object = nil
-	} else if val := msg.Object.Fields["time"].GetStringValue(); val != "" {
-		if t, err := time.Parse(time.RFC3339Nano, val); err == nil {
-			msg.NativeTimestamp = timestamppb.New(t)
+	} else {
+		if val := msg.Object.Fields["time"].GetStringValue(); val != "" {
+			if t, err := time.Parse(time.RFC3339Nano, val); err == nil {
+				msg.NativeTimestamp = timestamppb.New(t)
+			}
+		}
+		if val := msg.Object.Fields["log"].GetStringValue(); val != "" {
```

Comment:
What is the "log" field?  I think you're hitting the case where Loki is misconfigured and responses are wrapped inside this: https://docs.docker.com/config/containers/logging/json-file/

We need to handle this higher up, when we first get the line back from Loki.  It also means you're using a weird dev environment if you're running into this.